frontend/Figma UI Files/src/components/Navbar.tsx

```
@@ -0,0 +1,273 @@
+import React, { useState } from 'react';
+import { AppBar, Toolbar, Typography, InputBase, Badge, Avatar, IconButton, Card } from '@mui/material';
+import { Button } from './ui/button';
+import { Input } from './ui/input';
+import { Badge } from './ui/badge';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Badge'. Consider to delete it or rename it.

'Badge' is defined here:


(lint/suspicious/noRedeclare)